### ISSUE TITLE
lesion-predict-image error 

### ISSUE THREAD

**Issue by AliceSantilli** [jcreinhold/tiramisu-brulee#18]
## 🐛 Bug

lesion-predict-image shouldn't require the user to specify `subject`.

```python
Traceback (most recent call last):
  File "/home/ec2-user/anaconda3/envs/pytorch_latest_p37/bin/lesion-predict-image", line 8, in <module>
    sys.exit(predict_image())
  File "/home/ec2-user/anaconda3/envs/pytorch_latest_p37/lib/python3.7/site-packages/tiramisu_brulee/experiment/cli/predict.py", line 188, in predict_image
    _predict(args, parser, False)
  File "/home/ec2-user/anaconda3/envs/pytorch_latest_p37/lib/python3.7/site-packages/tiramisu_brulee/experiment/cli/predict.py", line 263, in _predict
    _predict_whole_image(args, model_path, model_num)
  File "/home/ec2-user/anaconda3/envs/pytorch_latest_p37/lib/python3.7/site-packages/tiramisu_brulee/experiment/cli/predict.py", line 201, in _predict_whole_image
    dm = LesionSegDataModulePredictWhole.from_csv(**dict_args)
  File "/home/ec2-user/anaconda3/envs/pytorch_latest_p37/lib/python3.7/site-packages/tiramisu_brulee/experiment/data.py", line 771, in from_csv
    check_dicom=check_dicom,
  File "/home/ec2-user/anaconda3/envs/pytorch_latest_p37/lib/python3.7/site-packages/tiramisu_brulee/experiment/data.py", line 1105, in csv_to_subjectlist
    df = pd.read_csv(filename, index_col="subject")
  File "/home/ec2-user/anaconda3/envs/pytorch_latest_p37/lib/python3.7/site-packages/pandas/io/parsers.py", line 610, in read_csv
    return _read(filepath_or_buffer, kwds)
  File "/home/ec2-user/anaconda3/envs/pytorch_latest_p37/lib/python3.7/site-packages/pandas/io/parsers.py", line 468, in _read
    return parser.read(nrows)
  File "/home/ec2-user/anaconda3/envs/pytorch_latest_p37/lib/python3.7/site-packages/pandas/io/parsers.py", line 1057, in read
    index, columns, col_dict = self._engine.read(nrows)
  File "/home/ec2-user/anaconda3/envs/pytorch_latest_p37/lib/python3.7/site-packages/pandas/io/parsers.py", line 2101, in read
    values = data.pop(self.index_col[i])
KeyError: 'subject'
```

**Comment by AliceSantilli** [comment]
lesion-predict-image --model-path pet_and_noAC_exp01.ckpt --t1 "38132639/34430964/D2021_04_30/PT/S0501_Scene_tejan/S0006_PET-AC, AX, Ga68 DOTATATE.nii.gz" --out predictions/noAC/38132639/34430964/D2021_04_30/PT/S0501_Scene_tejan/img_0001_01089_S2S.nii.gz

**Comment by jcreinhold** [comment]
I suspect this is due to a multiprocessing issue and can be resolved by setting `-nw 0` in the CLI call. Otherwise, this is likely fixed with aa705bd53210867356c0950d4b9303a78326c34d (this update isn't in v0.1.31, but will be in the next patch version; the easiest way to get this update is to clone the repo and install it with `python setup.py develop`). Reopen if the issue persists.
